### ISSUE TITLE
WandB logger showing `global_step` instead of `epochs`

### ISSUE THREAD

**Issue by sgalpha01** [ashleve/lightning-hydra-template#581]
I am using your template for a project, and I find it incredibly useful. Currently, I have integrated wandb as the logger, utilizing the default config file for logger settings. The functionality is working perfectly, but I have encountered a small problem.

In the wandb dashboard, the x-axis has `global_step` instead of `epochs`. I would like to know if there is a way to log `epoch` without modifying the code directly, but by making alterations solely within the config files.

I am attaching a screenshot for your reference:
![image](https://github.com/ashleve/lightning-hydra-template/assets/42481964/c2de3afa-f482-4a14-95f9-4222733ce3d1)

Config:
```yaml
# https://wandb.ai

wandb:
  _target_: lightning.pytorch.loggers.wandb.WandbLogger
  # name: "" # name of the run (normally generated by wandb)
  save_dir: "${paths.output_dir}"
  offline: False
  id: null # pass correct id to resume experiment!
  anonymous: null # enable anonymous logging
  project: "DeepDTA"
  log_model: False # upload lightning ckpts
  prefix: "" # a string to put at the beginning of metric keys
  # entity: "" # set to name of your wandb team
  group: ""
  tags: []
  job_type: ""
```

**Comment by one2clouds** [comment]
Hello! So have you solved the issue then? What to do to log with epoch instead of logging with steps?
